lib/HLSL/HLOperationLower.cpp

```diff
@@ -456,16 +455,16 @@ Value *TrivialDxilOperation(Function *dxilFunc, OP::OpCode opcode,
       retVal = Builder.CreateInsertElement(retVal, EltOP, i);
     }
     return retVal;
-  } else {
-    if (!RetTy->isVoidTy()) {
-      Value *retVal =
-          Builder.CreateCall(dxilFunc, args, hlslOP->GetOpCodeName(opcode));
-      return retVal;
-    } else {
-      // Cannot add name to void.
-      return Builder.CreateCall(dxilFunc, args);
-    }
   }
+
+  // Cannot add name to void.
+  if (RetTy->isVoidTy()) {
+    return Builder.CreateCall(dxilFunc, args);
+  }
```

Comment:
```suggestion
  // Cannot add name to void.
  if (RetTy->isVoidTy()) 
    return Builder.CreateCall(dxilFunc, args);
```

https://llvm.org/docs/CodingStandards.html#don-t-use-braces-on-simple-single-statement-bodies-of-if-else-loop-statements